.ci/pipeline_definitions

```diff
@@ -14,6 +14,13 @@ machine-controller-manager-provider-openstack:
             image: 'eu.gcr.io/gardener-project/gardener/machine-controller-manager-provider-openstack'
             target: machine-controller
   jobs:
+    create-upgrade-prs:
+      traits:
+        component_descriptor: ~
+        version: ~
+        cronjob:
+          interval: '5m'
```

Comment:
Vedran suggested to update this interval to `24h` , I did the change in one provider others are left though